README.md

```diff
@@ -113,6 +113,10 @@ The default generator will run the `install`, `shop`, and `home_controller` gene
 ```sh
 $ rails generate shopify_app --api_key <your_api_key> --secret <your_app_secret>
 ```
+or optionally with an additional application_name argument as:
+```sh
+$ rails generate shopify_app --application_name <your_app_name> --api_key <your_api_key> --secret <your_app_secret> 
+```
```

Comment:
Can you remove this example. The new param is still listed as an option below and I think that is enough